.cargo/config.toml

```diff
@@ -0,0 +1,90 @@
+[target.'cfg(all())']
```

Comment:
Initially, I thought, "Neat trick!" But then I wondered, since it's supposed to be enabled all the time, can't it just use:

```toml
[build]
rustflags = [
  ...
]
```

Or is there some nuance here I'm not aware of?

Reply:
Yeah, it's supposed to prevent overriding other `RUSTFLAGS`.